package.nls.json

```diff
@@ -6,13 +6,15 @@
     "AWS.channel.aws.toolkit": "AWS Toolkit",
     "AWS.codelens.lambda.invoke": "Run Locally",
     "AWS.codelens.lambda.invoke.debug": "Debug Locally",
+    "AWS.configuration.title": "AWS Configuration",
     "AWS.configuration.regionDescription": "The regions to query for resources to display in the Explorer.",
     "AWS.configuration.profileDescription": "The name of the credential profile to obtain credentials from.",
     "AWS.configuration.description.onDefaultRegionMissing": "Action to take when a Profile's default region is hidden in the Explorer. Possible values:\n* `add` - shows region in the explorer\n* `ignore` - does nothing with the region\n* `prompt` - (default) asks the user what they would like to do.",
     "AWS.configuration.description.recentCredentials": "Credentials most recently used to connect to AWS",
     "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time in milliseconds to wait for local lambda inkokes to start before attempting to attach the debugger.",
     "AWS.configuration.description.samcli.location": "Location of SAM CLI. SAM CLI is used to create, build, package, and deploy Serverless Applications. [Learn More](https://aws.amazon.com/serverless/sam/)",
-    "AWS.configuration.title": "AWS Configuration",
+    "AWS.configuration.description.telemetry": "Enable anonymous usage data about the toolkit to be sent to AWS. The option `\"null\"` respects the value specified in `telemetry.enableTelemetry`",
```

Comment:
Consider:

from
The option `\"null\"` respects the value specified in `telemetry.enableTelemetry`

to
The option `\"null\"` respects the VS Code based value specified in `telemetry.enableTelemetry`

Also, instead of a "Nullable Bool", what if you made a custom string enum containing:
* "Enable"
* "Disable"
* "Use the same setting as VS Code"

Something like this might be more intuitive, not requiring an explanation of what null means. I like the idea of referencing `telemetry.enableTelemetry` regardless.

Reply:
:+1: Changing the display text and enum